build.gradle

```diff
@@ -1,21 +1,20 @@
 plugins {
-    id "org.jetbrains.kotlin.jvm" version "1.3.50"
+    id "org.jetbrains.kotlin.jvm" version "1.5.31"
 }
 
 group = 'com.anaplan.engineering'
-version = '2.6.4.25'
+version = '3.0.2'
 
-apply plugin: 'kotlin'
 apply plugin: 'java-gradle-plugin'
 apply plugin: 'maven-publish'
 
 ext {
-    kotlinVersion = "1.3.50"
-    overtureVersion = "2.6.4"
+    kotlinVersion = "1.5.31"
```

Comment:
could you switch to use Kotlin platform while we're here, e.g. https://github.com/anaplan-engineering/vdm-pretty-printer/blob/main/build.gradle#L31